infra/terraform/test_fixtures/terraform-google-vm.tf

```diff
@@ -0,0 +1,99 @@
+
+cals {
```

Comment:
```suggestion
locals {
```

Reply:
Thank you and my apologies for this error.